Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/.eslintrc.js

```diff
@@ -8,11 +8,11 @@ module.exports = {
       'plugin:cypress/recommended',
     ],
     "rules": {
-        "no-mixed-spaces-and-tabs": 0, // disable rule
-        "no-constant-condition": 0,
-        "no-unused-vars": 0,
-        "no-undef": 0,
-        "no-useless-escape": 0,
-        "cypress/no-assigning-return-values": 0
+        "no-mixed-spaces-and-tabs": 1, // disable rule
```

Comment:
The comment says disabled, but the `1` says it's enabled.

Reply:
thanks.

Reply:
I've addressed this now.